relay-server/src/services/buffer/envelope_store/sqlite.rs

```diff
@@ -353,8 +353,8 @@ impl SqliteEnvelopeStore {
             .shared_cache(true);
 
         let db = SqlitePoolOptions::new()
-            .max_connections(config.spool_envelopes_max_connections())
-            .min_connections(config.spool_envelopes_min_connections())
+            .max_connections(1)
+            .min_connections(0)
```

Comment:
Are you sure we want `0` as min? I would expect `1` otherwise we won't be able to connect I assume.

Reply:
0 Is the default if you do not specify `min_connections` at all. But we previously used 1 in all our configs so I will put 1 here as well 👍.